SPECS/pytz/pytz.spec

```diff
@@ -42,16 +41,31 @@ Library Reference (``datetime.tzinfo``).
 %py3_install
 
 %check
-PATH=%{buildroot}%{_bindir}:${PATH} \
-PYTHONPATH=%{buildroot}%{python3_sitelib} \
-    py.test%{python3_version} -v
+pip3 install --upgrade pip wheel flake8
+pushd pytz/tests
+check_status=0
```

Comment:
Please add "CHECK START" and "CHECK DONE" echo statements. Ptest parsing scripts rely on them.

Reply:
The are added automatically added by `rpmbuild`